PortalGenius.WPF/ApiStartup.cs

```diff
@@ -21,9 +23,23 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            var configuration = new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).AddJsonFile("appsettings.json").Build();
+
             services.AddDbContext<AppDbContext>(options =>
             {
-                options.UseSqlite(Configuration.GetConnectionString("Sqlite"));
+                switch (configuration.GetSection("DatabaseInUse").Value)
+                {
+                    case "Sqlite":
+                        options.UseSqlite(Configuration.GetConnectionString("Sqlite"));
+                        break;
+                    case "MSSQL":
+                        break;
+                    case "PostgreSQL":
+                        break;
+                    case "Oracle":
+                        break;
+                    default: throw new ArgumentException("database_in_use missing from appsettings.json");
```

Comment:
`database_in_use` zou ik vervangen met `DatabaseInUse`, omdat deze ook zo in de config staat. Dan weet je ook gelijk welke property je moet veranderen

Reply:
👍